bin/fetch

```diff
@@ -22,6 +22,11 @@ use Getopt::Long::Descriptive;
 use Open311::GetServiceRequests;
 use Open311::GetServiceRequestUpdates;
 
```

Comment:
Not sure if this is needed, but the script was failing on 'Can't call method "get" on an undefined value at /Users/victoria/work/my-society/fixmystreet-local/commonlib/perllib/mySociety/Locale.pm line 22.' after adding my changes 🤔

Reply:
Can't hurt! You're saying it didn't do it without your changes? Not sure why you would get this - it means that translations haven't been set up and it's trying to translate something, but nothing you've changed should involve a translation?

Reply:
Was it actually fetching an update when giving this error?

Reply:
Hmm, I'll have another look at this - will try to get the other fixes for this PR done first :-)